framework/src/play/mvc/Http.java

```diff
@@ -1018,4 +1022,20 @@ public WebSocketFrame(byte[] data) {
 
     public static class WebSocketClose extends WebSocketEvent {
     }
+
+    public enum SAMESITE {
```

Comment:
It's probably a matter of taste, but I would prefer `SameSite` name. Upper case doesn't make it readable.